src/scripts/utilities/Sound.ts

```diff
@@ -15,9 +16,17 @@ class Sound {
             this.initialized = true;
             // This is needed to be able to play sounds on mobile devices
             $(document).one('click', () => {
-                this.sound.play();
-                this.sound.pause();
-                this.sound.src = src;
+                this.sound.play().then(() => {
+                    this.sound.pause();
+                    this.sound.src = src;
+                }).catch(() => {
+                    this.sound.pause();
+                    this.sound.src = src;
+                });
```

Comment:
Thanks, forgot to look up if there was a finally method on promises before pushing.

I've added the timeout as well, because it was still sometimes not loading after the promise.